src/node/CurveNode.ts

```diff
@@ -0,0 +1,36 @@
+import { Mode } from '@/types';
+
+import PathNode from './PathNode';
+
+export default class CurveNode extends PathNode {
+    x1: number;
+    y1: number;
+    x2: number;
+    y2: number;
+    x3: number;
+    y3: number;
+
+    constructor(
+        mode: Mode,
+        x1: number,
+        y1: number,
+        x2: number,
+        y2: number,
+        x3: number,
+        y3: number
```

Comment:
w sumie, jak jest tyle parametrów to lubię to przenieść do obiektu, ale tutaj kwestia z czego by się wygodniej korzystało, skoro to ma być taki deklaratywny sposób budowania czegoś 🤔